src/parameter.jl

```diff
@@ -73,6 +73,15 @@ Base.:(==)(p1::Number, p2::Parameter) = eqeq(p2, p1)
 # following the same convention in Phase.jl implementation
 # comparison have inconsistent, we are comparing phases to numbers
 # if cause trouble, will change
+#
+function contains(p::Parameter, θ::Symbol)
```

Comment:
Maybe you can overload the `Base.contains` method here

Reply:
Thanks! I will overload it.